templates/daapd/meta.yaml

```diff
@@ -0,0 +1,70 @@
+name: Daapd
+description:
+  Daapd is a robust, open-source DAAP (Digital Audio Access Protocol) server
+  that allows users to share music libraries over a network. It is designed to
+  work seamlessly with iTunes, Roku SoundBridges, and other DAAP clients,
+  providing an efficient and reliable way to stream audio content across
+  devices. Daapd also supports streaming internet radio, podcasts, and AirPlay
+  devices. With its powerful backend, Daapd ensures smooth performance, a wide
+  range of supported formats, and advanced features like library management and
+  transcoding.
+instructions: null
+changeLog:
+  - date: 2024-12-26
+    description: First Release
+links:
+  - label: Website
+    url: https://github.com/ejurgensen/forked-daapd
+  - label: Documentation
+    url: https://ejurgensen.github.io/forked-daapd/
+  - label: Github
+    url: https://github.com/ejurgensen/forked-daapd
+contributors:
+  - name: Ahson Shaikh
+    url: https://github.com/Ahson-Shaikh
+schema:
+  type: object
+  required:
+    - appServiceName
+    - appServiceImage
+  properties:
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: daapd
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: lscr.io/linuxserver/daapd:28.10.20241218
```

Comment:
sometimes you use lscr.io and sometimes you omit it

for linuxserver images you can always use it, it's a proxy/mirror and helps users avoid docker hub rate limits